python_modules/dagstermill/dagstermill/__init__.py

```diff
@@ -255,14 +339,35 @@ def populate_context(dm_context_data):
         dm_context_data['marshal_dir'],
         dm_context_data['environment_config'],
         dm_context_data['output_log_path'],
+        dm_context_data['input_name_type_dict'],
+        dm_context_data['output_name_type_dict'],
     )
 
 
 def load_parameter(input_name, input_value):
     check.invariant(MANAGER_FOR_NOTEBOOK_INSTANCE.populated_by_papermill, 'populated_by_papermill')
-    solid_def = MANAGER_FOR_NOTEBOOK_INSTANCE.solid_def
-    input_def = solid_def.input_def_named(input_name)
-    return read_value(input_def.runtime_type, input_value)
+    if MANAGER_FOR_NOTEBOOK_INSTANCE.solid_def is None:
+        check.invariant(
+            MANAGER_FOR_NOTEBOOK_INSTANCE.input_name_type_dict,
```

Comment:
i think this will throw if `MANAGER_FOR_NOTEBOOK_INSTANCE.input_name_type_dict` is `{}`, which is probably not what we want

Reply:
Changed the condition to explicitly be 'obj is not None'--good catch